packages/button/index.tsx

```diff
@@ -22,11 +22,13 @@
 
 import React from 'react';
 import classnames from 'classnames';
-import {withRipple, InjectedProps} from '@material/react-ripple';
+import {withRipple, InjectedProps, RippledComponentProps} from '@material/react-ripple';
 
 import {CSS_CLASSES} from './constant';
 
 type ButtonTypes = HTMLAnchorElement | HTMLButtonElement;
+// TODO(https://github.com/material-components/material-components-web-react/pull/953)
+export type removeOnceTypeResolutionIsFixed = RippledComponentProps<ButtonTypes>;
```

Comment:
Types should be capitalized: `remove` --> `Remove`.

Also please create an issue to remove this when the TS version is update. Its not clear to me what we're waiting for, or if this is a new TS bug.